server/src/assembly/resources/web/src/Table.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react'
+import './Table.css'
+
+class Table extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+          lists: [],
+          active: false
+        };
+        this.getMachineAction = this.getMachineAction.bind(this);
+        this.toggleClass = this.toggleClass.bind(this);
+        this.toggleContent = this.toggleContent.bind(this);
+    }
+
+    async getMachineAction() {
+        try {
+            await fetch( 'http://localhost:8181/rest/sql_arguments', {
```

Comment:
Hardcode here? If I change the http_port=8181 property in iotdb-engine.properties, can the bundle.js be built correctly?